src/components/card/card-test.stories.tsx

```diff
@@ -26,44 +26,58 @@ export default {
     },
   },
   argTypes: {
-    action: { action: "action" },
+    children: { table: { disable: true } },
+    onClick: { table: { disable: true } },
+    footer: { table: { disable: true } },
   },
 } as ComponentMeta<typeof Card>;
 
-export const DefaultStory: ComponentStory<typeof Card> = (
-  args: Partial<CardProps>
+export const DefaultStory = (
+  args: Omit<CardProps, "onClick" | "children" | "footer">
 ) => {
   return (
-    <Card interactive {...args}>
+    <Card
+      {...args}
+      footer={
+        <CardFooter>
+          <CardColumn>
+            <Link icon="link" href="https://carbon.sage.com/">
+              Footer link
+            </Link>
+          </CardColumn>
+        </CardFooter>
+      }
+    >
       <CardRow>
         <CardColumn align="left">
-          <Heading title="Stripe - [account name]" divider={false} />
-          <Typography variant="h5">user.name@sage.com</Typography>
+          <Heading title="Heading" divider={false} />
+          <Typography fontSize="16px" m={0}>
+            Additional text
+          </Typography>
         </CardColumn>
         <CardColumn align="right">
           <Icon type="image" />
         </CardColumn>
       </CardRow>
       <CardRow>
         <CardColumn>
-          <Typography variant="h5" fontWeight="bold">
-            Stripe Balance
+          <Typography fontSize="16px" m={0} fontWeight="bold">
+            Body text
           </Typography>
-          <Heading title="£ 0.00" divider={false} />
-          <Typography fontSize="12px">LAST ENTRY: 15 DAYS AGO</Typography>
+          <Heading title="More text" divider={false} />
+          <Typography>Even more text</Typography>
         </CardColumn>
       </CardRow>
-      <CardFooter>
-        <CardColumn>
-          <Link icon="link" href="https://carbon.sage.com/">
-            View Stripe Dashboard
-          </Link>
-        </CardColumn>
-      </CardFooter>
     </Card>
   );
 };
 DefaultStory.storyName = "default";
+DefaultStory.argTypes = {
+  locale: {
+    options: ["en", "fr", "no", "es-ES", "pt-PT", "it"],
```

Comment:
**question**: I'm not sure I understand the context behind the `locale` arg here, would you be able to clarify this please?

Reply:
This looks incorrect, IIRC I was trying to get the controls to update but realised you need to rebuild storybook but forgot to delete this